enterprise/server/tasksize/tasksize.go

```diff
@@ -121,7 +121,11 @@ func (s *taskSizer) Get(ctx context.Context, task *repb.ExecutionTask) *scpb.Tas
 	if !*useMeasuredSizes {
 		return nil
 	}
-	props := platform.ParseProperties(task)
+	props, err := platform.ParseProperties(task)
+	if err != nil {
+		log.CtxWarningf(ctx, "Failed to parse task properties: %s", err)
```

Comment:
let's switch these to `log.Infof()` since they are user errors - I think we are starting to move towards treating Warnings as real issues, e.g. w/ the app canary process

also can you add a TODO to reject the task in this case? we probably shouldn't accept tasks that fail validation, so that users can catch errors soooner.

Reply:
fixed